lib/generators/rails/policy/templates/policy.rb.tt

```diff
@@ -0,0 +1,52 @@
+<%- module_namespacing do -%>
+class <%= class_name %>Policy < ApplicationPolicy
+<%- if options[:headless] -%>
+  # Headless policy (used when no corresponding ActiveRecord/Model):
+  # - use `authorize(:<%= singular_name %>, :action?)` in controller actions
+  # - see https://github.com/varvet/pundit#headless-policies
+  def initialize(user, _record)
+    @user = user
+  end
+<%- else -%>
+  class Scope < ApplicationPolicy::Scope
+    def resolve
+      case user
+      when CasaAdmin, Supervisor
+        scope.where(casa_org: @user.casa_org)
+      when Volunteer
+        # REMOVE IF NOT APPLICABLE (just an example, doesn't work for all cases)
+        # scope.assigned_to_user(@user)
+        scope.none
+      else
+        scope.none
+      end
+    end
+  end
+<%- end -%>
+<%- user_example = "is_admin? || is_supervisor?" -%>
+<%- record_example = "admin_or_supervisor_same_org?" -%>
+<%- if actions.empty? -%>
+
+  # No actions specified, Example usage:
+  # def index?
+  #   <%= user_example %>
+  # end
+  #
+  # def show?
+  #   <%= record_example %>
+  # end
+<%- end -%>
+<%- actions.each do |action| -%>
```

Comment:
@elasticspoon `actions` here is the full CRUD set for a scaffold generator, but if you use the `rails g controller Model index show` it would be ['index', 'show'], so only creates those policies.  if no actions it will make the comment above.

Reply:
oh, I changed the defaults, so that comment isn't happening for `rails g policy Thing `. I did that so it would work for a scaffold. hmm... gonna see if I can get that back...

Reply:
updated. 


When no actions, this comment is generated, plus example specs for index & show, with comment to modify/remove.